src/utils/trust/individual.trustee.mapper.ts

```diff
@@ -85,6 +96,20 @@ export const mapIndividualTrusteeByIdFromSessionToPage = (
 export const mapIndividualTrusteeFromSessionToPage = (
   trustee: Trust.IndividualTrustee,
 ): Page.IndividualTrusteesFormCommon => {
+
+  let stillInvolvedInTrust: string;
+  switch (trustee.still_involved) {
```

Comment:
it might be better to use an enum

Reply:
I agree. These "Yes", "No, "0" and "1" strings are repeated throughout the code-base.

I'd suggest we leave as is for now but log this as tech-debt somewhere (a JIRA ticket, maybe). @dlloyd1-ch ?

Reply:
The enum could just be local to this function for now. Then we'd have an example in the code that could be used elsewhere in future.

Reply:
Changed this.